microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -22,7 +25,7 @@
 from lisa.features import SerialConsole
 from lisa.operating_system import Redhat
 from lisa.sut_orchestrator.azure.tools import Waagent
-from lisa.tools import Dmesg, Echo, KdumpBase, KernelConfig, Lscpu, Stat
+from lisa.tools import Cat, Echo, KdumpBase, KernelConfig, Lscpu, Stat
```

Comment:
Why not use dmesg?

Reply:
The reason should be added in the commit message.

Reply:
Added.